docs/source/components/authorize.mdx

```diff
@@ -19,12 +19,12 @@ import Authorize from '@availity/authorize';
 
 <Authorize permissions="1234">
   {/* stuff to render if the user is authorized with permission indicated above */}
-</Authorize>
+</Authorize>;
```

Comment:
@shkn-not-strrd did a lot of work to get rid of these. don't make her have to do it again 😆